lib/context.js

```diff
@@ -92,6 +92,9 @@ Context.prototype.done = function(err, res) {
     }
 
     try {
+      this.res.setHeader("Cache-Control", "no-cache, no-store, must-revalidate");
+      this.res.setHeader("Pragma", "no-cache");
+      this.res.setHeader("Expires", "0");
```

Comment:
Shall we use ```this.res. getHeader``` to check a value have been set or not?

That can increase the flexibility of dpd-module / dpd-plugin